spec/queries/drug_stocks_query_spec.rb

```diff
@@ -76,6 +79,7 @@
         for_end_of_month: for_end_of_month)
 
       other_facility_patients = create_list(:patient, 1, registration_facility: other_facility, registration_user: user)
+      refresh_views
```

Comment:
DrugStock calls will now depend upon mat views, so adding the refresh views here.